src/localization/fr/general.json

```diff
@@ -0,0 +1,35 @@
+{
+  "general.prev": "Précédent",
+  "general.next": "Suivant",
+  "general.result": "Résultat",
+  "general.regex": "Regex",
+  "general.text": "Texte",
+  "general.start": "Démarrer",
+  "general.continue": "Continuer",
+  "general.completedStep": "Etape terminé!",
```

Comment:
would modify line 9 to 
  "general.completedStep": "Etape terminée!",

Reply:
Thanks for the review @MathisChartouni  :)